### PR TITLE
getdel

### PR DESCRIPTION
**Pull request by AvitalFineRedis**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `$ tox` pass with this change (including linting)?
- [x] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [x] Is the new or changed code fully tested?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change

Add support to GETDEL

### CODE REVIEW

**Comment by AvitalFineRedis** [comment]
@chayim

File: tests/test_commands.py [hunk 1]
```diff
@@ -727,6 +727,13 @@ def test_get_and_set(self, r):
         assert r.get('integer') == str(integer).encode()
         assert r.get('unicode_string').decode('utf-8') == unicode_string
 
+    @skip_if_server_version_lt('6.2.0')
+    def test_getdel(self, r):
+        assert r.getdel('a') is None
+        assert r.set('a', 1)
```
Comment (chayim) [inline tests/test_commands.py:733]: You shouldn't need to assert the set

**Comment by AvitalFineRedis** [comment]
@chayim

**Comment by chayim** [comment]
@AvitalFineRedis There are some conflicts now. Can you merge master back in, and resolve conflicts?

**Review by chayim — APPROVED** [review]

**Pull request merged**